types/src/key.rs

```diff
@@ -197,6 +207,9 @@ impl Display for FromStrError {
                     error
                 )
             }
+            FromStrError::EraSummary(error) => {
+                write!(f, "era-summary from string error: {}", error)
```

Comment:
```suggestion
                write!(f, "era-summary-key from string error: {}", error)
```
(This will fix the failing test)